lms/sql_tasks/sql_query.py

```diff
@@ -58,3 +59,14 @@ def dump(self, indent=""):
             )
         text += f"\n\nTime: {self.duration}"
         return textwrap.indent(text, indent)
+
+    @staticmethod
+    def _clean_query(query: str) -> str:
+        """Clean any of the query's lines marked as sensitive."""
+        lines = query.split("\n")
+        for i, line in enumerate(lines):
+            # Heavy handed here, we much rather have false positives than negatives
+            if "secret" in line.lower():
+                lines[i] = re.sub(r"[^\s]", "*", line)
+
+        return "\n".join(lines)
```

Comment:
```suggestion
        return "\n".join([
            re.sub(r"[^\s]", "*", line) if "secret" in line.lower() else line
            for line in query.split("\n")
        ])
```

Generally enumerate and index on a list suggest a missing list comprehension?